engine/src/main/java/com/google/android/fhir/sync/upload/request/UploadRequestGenerator.kt

```diff
@@ -53,3 +62,17 @@ internal object UploadRequestGeneratorFactory {
         )
     }
 }
+
+internal sealed class UploadRequestGeneratorOutput(
```

Comment:
similar to patch, should we name this to UploadRequestMapping, as well as the inheriting data classes?
```suggestion
internal sealed class UploadRequestGeneratorOutput(
```

Reply:
Done